pkg/tests/apis/alerting/notifications/receivers/receiver_test.go

```diff
@@ -126,6 +127,266 @@ func TestIntegrationResourceIdentifier(t *testing.T) {
 	})
 }
 
+func TestIntegrationResourcePermissions(t *testing.T) {
```

Comment:
Let's add some comments for the test suite. At first glance, it looks like the existing one but this one focuses on

Reply:
823db3461ff81876d4738300c2d0824c6551c08e